contracts/MarginCalculator.sol

```diff
@@ -23,7 +23,10 @@ contract MarginCalculator {
 
     address public addressBook;
 
+    /// @dev decimals used by strike price and oracle price
     uint256 internal constant BASE = 8;
+
+    /// @dev FixedPoint 0
     FPI.FixedPointInt internal ZERO = FPI.fromScaledUint(0, BASE);
```

Comment:
Is this also a constant?

Reply:
it should be but we can't declare struct as constant in solidity.